electrum_nmc/electrum/gui/qt/configure_name_dialog.py

```diff
@@ -183,8 +191,40 @@ def update_and_broadcast(self, identifier, value, transfer_to):
             self.main_window.show_error(_("Error broadcasting update for ") + formatted_name + ": " + str(e))
             return
 
+    def update_value_from_ascii(self):
+        try:
+            value_ascii = self.ui.dataEdit.text()
+            value = name_from_str(value_ascii, Encoding.ASCII)
+            value_hex = name_to_str(value, Encoding.HEX)
+            self.ui.dataEditHex.setText(value_hex)
+            self.ui.labelSubmitHint.setText(self.SubmitHintText)
+            self.ui.btnDNSEditor.setDisabled(False)
+        except Exception as e:
+            self.ui.labelSubmitHint.setText(f"{e}")
+            self.ui.btnDNSEditor.setDisabled(True)
+
+    def update_value_from_hex(self):
+        try:
+            value_hex = self.ui.dataEditHex.text()
+            value = name_from_str(value_hex, Encoding.HEX)
+
+            self.ui.labelSubmitHint.setText(self.SubmitHintText)
+            self.ui.btnDNSEditor.setDisabled(False)
+
+            try:
+                value_ascii = name_to_str(value, Encoding.ASCII)
+                self.ui.dataEdit.setText(value_ascii)
+            except Exception as e:
+                # Make sure we are in the hex_tab
+                # DNS Editor uses ASCII, hence we diable the button
+                self.ui.tabWidget.setCurrentIndex(1)
+                self.ui.btnDNSEditor.setDisabled(True)
+
+        except Exception as e:
+            self.ui.btnDNSEditor.setDisabled(True)
+            self.ui.labelSubmitHint.setText(f"{e}")
```

Comment:
Can we disable the OK button here too? AFAICT if the hex value is invalid, clicking OK will cause an error.

Reply:
Alright, I've made the update. 
Do we need to disable the UI when updating from ASCII? Updating from ASCII shouldn't cause an exception